src/main/java/org/opentripplanner/raptor/rangeraptor/transit/AccessPaths.java

```diff
@@ -103,11 +118,13 @@ public List<RaptorAccessEgress> arrivedOnBoardByNumOfRides(int round) {
     return filterOnTimePenaltyLimitIfExist(arrivedOnBoardByNumOfRides.get(round));
   }
 
-  public int calculateMaxNumberOfRides() {
-    return Math.max(
-      Arrays.stream(arrivedOnStreetByNumOfRides.keys()).max().orElse(0),
-      Arrays.stream(arrivedOnBoardByNumOfRides.keys()).max().orElse(0)
-    );
+  public static int calculateMaxNumberOfRides(@Nullable AccessPaths paths) {
+    return paths == null
```

Comment:
I actually think you don't need the nullability or the null check since this will never be called with a null value.